colcon.pkg

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
Does this file support comments in the file? If so, it'd be great to leave a brief comment how this file is used for.

Reply:
It's YAML, so comments prefixed with `#` are acceptable here.